WreckEngine/Game.cpp

```diff
@@ -1,23 +1,38 @@
 #include "Game.h"
 #include <iostream>
 
-Game::Game(GLprogram prog) : shader(prog) { }
+Game::Game()
+    : currState(nullptr)
```

Comment:
but there's no good reason for this not to be single line, it's short and it's an initializer list. In this case, convention would be to keep it on one line. I'm fine with that most times.